### PR TITLE
Replaced deprecated wagtail.utils.pagination module with django core …

### PR DESCRIPTION
**Pull request by einsfr**
…functionality, added missed app requirement in testing project setttings.

### CODE REVIEW

**Comment by einsfr** [comment]
Requirements change: Django >=2.0 and Wagtail >=2.4 since 2.4 dropped support for Django 1.11

File: wagtailmodelchooser/views.py [hunk 1]
```diff
@@ -62,7 +62,8 @@ def chooser(request, app_label, model_name, filter_name=None):
             raise Http404
         qs = filter_func(qs)
 
-    paginator, page = paginate(request, qs, per_page=10)
+    paginator = Paginator(qs, per_page=10)
+    page = paginator.get_page(request.GET.get('p'))
```
Comment (seb-b) [inline wagtailmodelchooser/views.py]: can you change this to:

`page = paginator.page(request.GET.get('p', 1))`

**Comment by seb-b** [comment]
I'll do a review of the supported django versions on the next release, I should check to see if everything works up to wagtail 2.6

**Comment by einsfr** [comment]
Works fine with Django 1.11 now

File: wagtailmodelchooser/views.py (hunk 1 above)
Comment (einsfr) [reply wagtailmodelchooser/views.py]: Done

**Comment by talebbits** [comment]
Any update when this will be released ?

**Comment by seb-b** [comment]
Sorry about the delay. 2.2.2 is out with this fix included

**Pull request merged**